api/well-known/README.md

```diff
@@ -0,0 +1,74 @@
+# Discovery APIs
+The APIs described here allow a user of the veraison to obtain and view meta information about the veraison deployment. Information about deployment can be queried according to the service e.g. (provisioning or verification).
+
+The relevant resource is created using in response to a client `GET`. The queried information is then outputted as shown examples below.
+
+## Well know API: Provisioning service
+Each well-know provisioning session is associated to its own resource, with the
+following attributes:
+
+* The allowed provisioning mediatypes;
+* The version of the provisioning service.
+
+### Quering information about the provisioning service
+```text
+>> Request:
+  GET /.well-known/veraison/provisioning
+  Host: veraison.example
+  Accept: application/vnd.veraison.provisioning+json
+  Content-Type: application/json
+
+
+<< Response:
+  HTTP/1.1 200 OK
+  Content-format: application/vnd.veraison.provisioning-well-known-session+json
+
+  {
+  "media-types": [
+    "application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1",
+    "application/corim-unsigned+cbor; profile=http://enacttrust.com/veraison/1.0.0",
+    "application/corim-unsigned+cbor; profile=http://arm.com/cca/ssd/1"
+  ],
+  "version": "commit-cb11fa0"
+}
+```
+
+
+## Well know API: Verification service
+Each well-know verification session is associated to its own resource, with the
+following attributes:
+
+* The public key used to sign the evidence;
+* The allowed mediatypes for attestation evidence;
+* The version of the verification service.
+
+### Quering information about the verification service
+```text
+>> Request:
+  GET /.well-known/veraison/verification
+  Host: veraison.example
+  Accept: application/vnd.veraison.verification+json
+  Content-Type: application/json
+
+
+<< Response:
+  HTTP/1.1 200 OK
+  Content-format: application/vnd.veraison.verification-well-known-session+json
+
+  {
+  "public-key": {
```

Comment:
I would give this a more descriptive name. "public-key" doesn't really tell you how this should be used. Perhaps something like "result-validation-key"..?